deprecate chat order field (#687)
livechat/livechat-public-docs: content/messaging/customer-chat-api/changelog/index.mdx

- Method **List Chats** ([Web](/messaging/customer-chat-api/v3.2/#list-chats) & [RTM](/messaging/customer-chat-api/v3.2/rtm-reference/#list-chats)) is paginated with `page_id` now.
- The [Login](/messaging/customer-chat-api/v3.2/rtm-reference/#login) method was extended with the application info.
- The **Get Chat Threads** method evolved into **Get Chat** ([Web](/messaging/customer-chat-api/v3.2/#get-chat) & [RTM](/messaging/customer-chat-api/v3.2/rtm-reference/#get-chat)). It now allows to retrieve a chat containing a particular thread. It also changed its shape; it no longer contains the `order` field and the `chat` key.

> - `order` in `chat` object has been removed from the payload of **Incoming Chat** push. Also, this field has been replaced with `last_thread_created_at` in the response of **List Chats** method.

Reviewer: ```suggestion
- The `order` field in the `chat` object was removed from the payload of the **incoming_chat** push. Also, this field was replaced with `last_thread_created_at` in the response of the **List Chats** method.
```
